### PR TITLE
Adjustments to brms product overview page.

### PR DESCRIPTION
**Pull request by eschabell**
Please push these changes for Preview.

### CODE REVIEW

**Comment by ollyjshaw** [comment]
ok to test

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

File: products/brms/overview.adoc [hunk 1]
```diff
@@ -41,8 +42,8 @@ image:#{cdn(site.base_url + '/images/icons/products/products_bar_graph.png')}["B
...
+Learn how the Business Resource Planner can tackle a wide range of planning problems, from delivery routing to shift scheduling and even managing compute resources in the cloud. +
 Download JBoss BRMS and explore the tools for authoring business rules, provided as both standalone web applications and plug-ins for JBoss Developer Studio.
 
 [.large-12.columns]
-image:#{cdn(site.base_url + '/images/products/brms/capture.png')}["BRMS Screenshot", max-width=600]
+https://vimeo.com/album/2982444/video/143059283[image:#{cdn(site.base_url + '/images/products/brms/planner-video.png')}["Planner Video", max-width=600]]
```
Comment (paulrobinson) [inline products/brms/overview.adoc]: This should stay on-site.

If this video isn't already available on the site, please add it: https://github.com/redhat-developer/developers.redhat.com/blob/master/CONTRIBUTING.md#to-add-a-vimeo-video-to-the-site

Comment (eschabell) [inline products/brms/overview.adoc]: Video is on site, added to the account you pointed me to.

**Comment by LightGuard** [comment]
Why are we merging in cdn changes to this?

File: products/brms/overview.adoc (hunk 1 above)
Comment (LightGuard) [inline products/brms/overview.adoc]: This should use the same code for adding videos that we have in other parts of the site:

    #{video_thumb(video('https://vimeo.com/143059283'), 'video_thumb.html.slim')}

Comment (eschabell) [inline products/brms/overview.adoc]: Let me look at this next week then, I can adjust it. I was not aware of the
new line you mentioned there.

Regards,
Eric D. Schabell

**Comment by paulrobinson** [comment]
I'm seeing problems with the video here: http://it-developers-pr.stage.redhat.com/pr/605/build/199/products/brms/overview/

@LightGuard any ideas?

**Comment by LightGuard** [comment]
Looks like it needs to be in a pass through block:

    +++
    #{video_thumb(...)}
    +++

Because video_thumb returns html which asciidoctor is then escaping.

**Comment by eschabell** [comment]
@LightGuard I don't know what that means, but feel free to adjust until it works as you think is needed for video to appear instead of html.

**Comment by eschabell** [comment]
This has not appeared yet on production site.... what is the status? Need something from me?

**Comment by paulrobinson** [comment]
The fix that @LightGuard mentions needs making.

**Comment by eschabell** [comment]
Will @LightGuard be making that or do I need to figure out what goes in there? Trying to complete this pre-holiday shutdown.

**Comment by paulrobinson** [comment]
@LightGuard is pretty busy at the moment, so I suggest you do it yourself if you want it before the holidays.

**Comment by eschabell** [comment]
I already pushed these changes back when he asked....

**Comment by paulrobinson** [comment]
Take a look at: http://it-developers-pr.stage.redhat.com/pr/605/build/267/products/brms/overview/. The problem still seems to be happening.

**Comment by LightGuard** [comment]
Like my last comment said, you need to wrap the video code you already have
in

    +++
    +++

That's a pass through block in asciidoctor, meaning it'll take whatever is
given and pass it directly into the HTML as HTML. Since we're interpolating
the document, the function call, all the bits within `#{}`, will be
processed by awestruct before it's sent to asciidoctor for final
processing.

On Wednesday, December 16, 2015, Paul Robinson <notifications@github.com>
wrote:

> Take a look at:
> http://it-developers-pr.stage.redhat.com/pr/605/build/267/products/brms/overview/.
> The problem still seems to be happening.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-165234122>
> .
>


-- 
Sent from Gmail Mobile

**Comment by eschabell** [comment]
Thanks @LightGuard , done and pushed into branch.

**Comment by paulrobinson** [comment]
Seems to be working now. However, I think the layout needs some work.

See: https://www.dropbox.com/s/sf9nlyrtajo8nly/Screenshot%202015-12-19%2013.53.37.png?dl=0
On: http://it-developers-pr.stage.redhat.com/pr/605/build/281/products/brms/overview/

@adelasofia might have time to take a look if you are not sure how to improve it.

**Comment by eschabell** [comment]
Yeah, let me look at this monday and push a change or two as layout can be improved.

**Comment by eschabell** [comment]
Seems like a simple fix, removed second column, hope it runs the video under the text now.

**Comment by LightGuard** [comment]
No need for the image anymore is there?

On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com>
wrote:

> Seems like a simple fix, removed second column, hope it runs the video
> under the text now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171680>
> .
>


-- 
Sent from Gmail Mobile

**Comment by eschabell** [comment]
Yes. The image is the video. That won't be removed?

On Sunday, December 20, 2015, Jason Porter <notifications@github.com> wrote:

> No need for the image anymore is there?
>
> On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
>
> > Seems like a simple fix, removed second column, hope it runs the video
> > under the text now.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171680
> >
> > .
> >
>
>
> --
> Sent from Gmail Mobile
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171922>
> .
>


-- 
Regards,
Eric D. Schabell

**Comment by LightGuard** [comment]
The image is the video? No, we use a thumbnail from video service. Take a
look at the source when it builds, it won't be the image you added.

On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com>
wrote:

> Yes. The image is the video. That won't be removed?
>
> On Sunday, December 20, 2015, Jason Porter <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
> > No need for the image anymore is there?
> >
> > On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>>
> > wrote:
> >
> > > Seems like a simple fix, removed second column, hope it runs the video
> > > under the text now.
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171680
> > >
> > > .
> > >
> >
> >
> > --
> > Sent from Gmail Mobile
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171922
> >
> > .
> >
>
>
> --
> Regards,
> Eric D. Schabell
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166172108>
> .
>


-- 
Sent from Gmail Mobile

**Comment by eschabell** [comment]
Ash, yes the video thumb is fine. Shall I remove the image link?

On Sunday, December 20, 2015, Jason Porter <notifications@github.com> wrote:

> The image is the video? No, we use a thumbnail from video service. Take a
> look at the source when it builds, it won't be the image you added.
>
> On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
>
> > Yes. The image is the video. That won't be removed?
> >
> > On Sunday, December 20, 2015, Jason Porter <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>> wrote:
> >
> > > No need for the image anymore is there?
> > >
> > > On Sunday, December 20, 2015, Eric D. Schabell <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>
> > > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>');>>
> > > wrote:
> > >
> > > > Seems like a simple fix, removed second column, hope it runs the
> video
> > > > under the text now.
> > > >
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > <
> > >
> >
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171680
> > > >
> > > > .
> > > >
> > >
> > >
> > > --
> > > Sent from Gmail Mobile
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171922
> > >
> > > .
> > >
> >
> >
> > --
> > Regards,
> > Eric D. Schabell
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166172108
> >
> > .
> >
>
>
> --
> Sent from Gmail Mobile
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166172232>
> .
>


-- 
Regards,
Eric D. Schabell

**Comment by LightGuard** [comment]
Yes, you don't need the image anymore.

On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com>
wrote:

> Ash, yes the video thumb is fine. Shall I remove the image link?
>
> On Sunday, December 20, 2015, Jason Porter <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
> > The image is the video? No, we use a thumbnail from video service. Take a
> > look at the source when it builds, it won't be the image you added.
> >
> > On Sunday, December 20, 2015, Eric D. Schabell <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>>
> > wrote:
> >
> > > Yes. The image is the video. That won't be removed?
> > >
> > > On Sunday, December 20, 2015, Jason Porter <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>
> > > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>');>> wrote:
> > >
> > > > No need for the image anymore is there?
> > > >
> > > > On Sunday, December 20, 2015, Eric D. Schabell <
> > notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>
> > > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>');>
> > > > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>
> > > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> > <javascript:_e(%7B%7D,'cvml','notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>');>');>');>>
> > > > wrote:
> > > >
> > > > > Seems like a simple fix, removed second column, hope it runs the
> > video
> > > > > under the text now.
> > > > >
> > > > > —
> > > > > Reply to this email directly or view it on GitHub
> > > > > <
> > > >
> > >
> >
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171680
> > > > >
> > > > > .
> > > > >
> > > >
> > > >
> > > > --
> > > > Sent from Gmail Mobile
> > > >
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > <
> > >
> >
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166171922
> > > >
> > > > .
> > > >
> > >
> > >
> > > --
> > > Regards,
> > > Eric D. Schabell
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166172108
> > >
> > > .
> > >
> >
> >
> > --
> > Sent from Gmail Mobile
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166172232
> >
> > .
> >
>
>
> --
> Regards,
> Eric D. Schabell
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/redhat-developer/developers.redhat.com/pull/605#issuecomment-166186340>
> .
>


-- 
Sent from Gmail Mobile

**Comment by paulrobinson** [comment]
Let me know when this is ready for review. It's still looking strange at the moment: https://www.dropbox.com/s/d87t9wxi8dqte7d/Screenshot%202015-12-21%2012.41.28.png?dl=0

Notice the odd layout, position of the video duration and the 'author' label that doesn't mention an author.

**Comment by eschabell** [comment]
I don't have an author, it is red hat commissioned official video? I also just pushed a change to remove more column layouts, should be inline now I hope.

**Comment by paulrobinson** [comment]
The content is still looking bad to me and I don't think we should be pushing it live in it's current state.

See: http://it-developers-pr.stage.redhat.com/pr/605/build/298/products/brms/overview/. In particular it looks odd having:

1. an empty author. I suggest removing this
2. A duration label stuck to this side of the thumbnail
3. The thumbnail is above the title, implying that it's part of the previous section.
4. The 'added' date doesn't add much value. I would suggest removing it.

@adelasofia are you able to take a look at this and fix the layout?

**Comment by adelasofia** [comment]
@eschabell @paulrobinson I just created a PR with the style improvements.

https://github.com/eschabell/developers.redhat.com/pull/1

**Comment by paulrobinson** [comment]
@adelasofia it's probably easier if you just raise a PR directly on this repo, including Eric's commit(s) and your own. I'll close this PR and wait for yours. Please notify @eschabell in the PR so he can review your proposal too.

**Pull request closed without merging**